src/test/java/com/epam/prejap/teatrees/ExecutableJarIT.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.prejap.teatrees;
+
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.CodeSource;
+import java.util.List;
+import java.util.jar.JarFile;
+
+import static org.testng.Assert.*;
+
+@Test
+public class ExecutableJarIT {
+
+    private static final String MAIN_CLASS_FQN = TeaTrees.class.getName();
+    private static String CLASSPATH;
+    private static final String JAR_EXTENSION = "jar";
+
+    static {
+        CodeSource codeSource = TeaTrees.class.getProtectionDomain().getCodeSource();
+        File jarFile;
+        try {
+            jarFile = new File(codeSource.getLocation().toURI().getPath());
+            CLASSPATH = jarFile.getParentFile().getPath();
+        } catch (URISyntaxException e) {
+            e.getMessage();
```

Comment:
This line does absolutely nothing. (It invokes a method and ignores its result.)